src/Microsoft.TemplateEngine.Edge/DefaultPathInfo.cs

```diff
@@ -0,0 +1,141 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#nullable enable
+
+using System;
+using System.IO;
+using System.Runtime.InteropServices;
+using Microsoft.TemplateEngine.Abstractions;
+
+namespace Microsoft.TemplateEngine.Edge
+{
+    /// <summary>
+    /// Default implementation of <see cref="IPathInfo"/>. If custom settings location are not passed, the following locations to be used: <br/>
+    /// - user profile: $USERPROFILE% on Windows, %HOME% on Unix <br/>
```

Comment:
Maybe we should skip this implementation detail, since we might want to switch to using:
```
Environment.GetFolderPath(Environment.SpecialFolder.UserProfile)
```

Reply:
removed that line